README.md

```diff
@@ -883,7 +883,7 @@ ui.start('#firebaseui-auth-container', {
       provider: 'saml.myProvider',
       providerName: 'MySAMLProvider',
       // To override the full label of the button.
-      // fullLabel: 'Constractor Portal',
```

Comment:
I couldn't figure out if this was `Constructor`, `Contractor`, or something else...